com.behringer.XAirEdit.yml

```
@@ -1,6 +1,6 @@
 app-id: com.behringer.XAirEdit
 runtime: org.freedesktop.Platform
-runtime-version: "19.08"
+runtime-version: "20.08"
```

```suggestion
runtime-version: "21.08"
```

There's now 21.08 FYI.